### PR TITLE
feat: allow users to set descriptions

### PR DESCRIPTION
**Pull request by h4ck3rk3y**
- [x] needs docs!

### CODE REVIEW

File: docs/docs/api-reference/starlark-reference/plan.md [hunk 1]
```diff
@@ -82,6 +86,10 @@ all_services = plan.add_services(
         "example-datastore-server-1": datastore_server_config_1,
         "example-datastore-server-2": datastore_server_config_2,
     },
+
+    # A human friendly description for the end user of the package
+    # OPTIONAL (Default: Adding 'NUMBER_OF_SERVICES' services with names 'SERVICE_NAMES')
+    description = "adding a service"
```
Comment (leoporoli) [inline docs/docs/api-reference/starlark-reference/plan.md]: "adding services" maybe?

File: docs/docs/api-reference/starlark-reference/plan.md [hunk 2]
```diff
@@ -462,7 +495,11 @@ The `run_python` instruction executes a one-time execution task. It runs the Pyt
         # The feature is enabled by default with a default timeout of 180s
         # OPTIONAL (Default: "180s")
         wait="180s"
-    )
+
+        # A human friendly description for the end user of the package
+        # OPTIONAL (Default: Running Python script)
+        description = "Running Python script"
```
Comment (leoporoli) [inline docs/docs/api-reference/starlark-reference/plan.md]: you could use lowercase here to match the other examples

File: docs/docs/api-reference/starlark-reference/plan.md [hunk 3]
```diff
@@ -539,6 +576,10 @@ The `run_sh` instruction executes a one-time execution task. It runs the bash co
         # The feature is enabled by default with a default timeout of 180s
         # OPTIONAL (Default: "180s")
         wait="180s"
+
+        # A human friendly description for the end user of the package
+        # OPTIONAL (Default: Running sh script)
+        description = "Running sh script"
```
Comment (leoporoli) [inline docs/docs/api-reference/starlark-reference/plan.md]: you could use lowercase here to match the other examples

File: docs/docs/api-reference/starlark-reference/plan.md [hunk 4]
```diff
@@ -613,6 +658,10 @@ plan.stop_service(
     # The service name of the service to be stopped.
     # MANDATORY
     name = "my_service",
+
+    # A human friendly description for the end user of the package
+    # OPTIONAL (Default: Stopping service 'SERVICE_NAME')
+    description = "Stopping service"
```
Comment (leoporoli) [inline docs/docs/api-reference/starlark-reference/plan.md]: you could use lowercase here to match the other examples

File: docs/docs/api-reference/starlark-reference/plan.md [hunk 5]
```diff
@@ -635,6 +684,10 @@ artifact_name = plan.store_service_files(
     # If not specified, it will be auto-generated.
     # OPTIONAL
     name = "my-favorite-artifact-name",
+
+    # A human friendly description for the end user of the package
+    # OPTIONAL (Default: Storing files from service 'SERVICE_NAME' at path 'PATH' to files artifact with name 'ARTIFACT_NAME')
+    description = "Storing some files"
```
Comment (leoporoli) [inline docs/docs/api-reference/starlark-reference/plan.md]: you could use lowercase here to match the other examples

File: docs/docs/api-reference/starlark-reference/plan.md [hunk 6]
```diff
@@ -657,6 +710,10 @@ artifact_name = plan.upload_files(
     # If not specified, it will be auto-generated.
     # OPTIONAL
     name = "my-artifact",
+
+    # A human friendly description for the end user of the package
+    # OPTIONAL (Default: Uploading file 'PATH' to files artifact 'ARTIFACT_NAME')
+    description = "Uploading file"
```
Comment (leoporoli) [inline docs/docs/api-reference/starlark-reference/plan.md]: you could use lowercase here to match the other examples

File: docs/docs/api-reference/starlark-reference/plan.md [hunk 7]
```diff
@@ -711,6 +768,10 @@ recipe_result = plan.wait(
     # Follows Go "time.Duration" format https://pkg.go.dev/time#ParseDuration
     # OPTIONAL (Default: "10s")
     timeout = "5m",
+
+    # A human friendly description for the end user of the package
+    # OPTIONAL (Default: Waiting for at most 'TIMEOUT' for service 'SERVICE_NAME' to reach a certain state)
+    description = "Waiting for a certain state"
```
Comment (leoporoli) [inline docs/docs/api-reference/starlark-reference/plan.md]: you could use lowercase here to match the other examples

**Review by leoporoli — APPROVED** [review]

**Review by tedim52 — APPROVED** [review]

**Pull request merged**